transformations/underscore_trick/README.md

```diff
@@ -0,0 +1,20 @@
+# Underscore Trick
+This perturbation adds noise to all types of text sources (sentence, paragraph, etc.).
+
+Author name: Marco Di Giovanni
+Author email: marco.digiovanni@polimi.it
+Author Affiliation: Politecnico di Milano and University of Bologna
+
+## What type of a transformation is this?
+This transformation acts like a perturbation to test robustness.
+It replaces some random spaces with underscores (or even other selected symbols).
+This transformation mimics particular behaviours:
+- names of folders, files, classes, functions (e.g., underscore_trick)
+- trick word counters when there is a limited maximum number of words to insert in an online questionary/quiz.
```

Comment:
Adding on to ahoimarie's point here, along with getting clarity / readability, I also think this application of tricking word limit counters is an interesting point and can be elaborated upon. 
Apart from underscore, you could perhaps provide examples of symbols like '@' or '-' hyphen, etc?